profiles/schema.py

```diff
@@ -516,6 +526,7 @@ class Input:
     def mutate_and_get_payload(cls, root, info, **input):
         profile_data = input.pop("profile")
         youth_profile_data = profile_data.pop("youth_profile", None)
+        sensitive_data = profile_data.pop("sensitivedata", None)
```

Comment:
Super minor nitpick, on `CreateProfileMutation` this variable is called `sensitivedata`.

Reply:
No it's a good nitpick 👍. Actually for historical reasons there are some inconsistencies with the naming (`sensitivedata` vs. `sensitive_data`, I'd prefer the latter which would become `sensitiveData` in the API). Maybe I should write a ticket to fix the naming to consistent. It's better to do in a separate ticket because it might break the API.